e2e/src/editCra.e2e-spec.ts

```diff
@@ -0,0 +1,152 @@
+import { EditCraPage } from './editCra.po';
+
+import { browser, by, ElementFinder } from 'protractor';
+
+describe('When I input my timesheet', () => {
+  let editCraPage: EditCraPage,
+      nameInput: ElementFinder,
+      emailInput: ElementFinder,
+      titleInput: ElementFinder,
+      clientInput: ElementFinder;
+
+  beforeEach(() => {
+    editCraPage = new EditCraPage();
+    editCraPage.navigateTo();
+    browser.waitForAngular();
+
+    nameInput = editCraPage.getInputByLabelText('Nom');
+    emailInput = editCraPage.getInputByLabelText('Email');
+    titleInput = editCraPage.getInputByLabelText('Intitulé');
+    clientInput = editCraPage.getInputByLabelText('Client final');
+  });
+
+  it('should be possible to enter my name', () => {
+    const name = 'Tester';
+          nameInput.sendKeys(name);
+
+    expect(nameInput.getAttribute('value')).toBe(name);
+  });
+
+  it('should be possible to enter my email', () => {
+    const email = 'tester@test.com';
+          emailInput.sendKeys(email);
+
+    expect(emailInput.getAttribute('value')).toBe(email);
+  });
+
+  it('should be possible to enter a mission title', () => {
+    const title = 'Testing';
+          titleInput.sendKeys(title);
+
+    expect(titleInput.getAttribute('value')).toBe(title);
+  });
+
+  it('should be possible to enter the client\'s name', () => {
+    const client = 'Test.com';
+          clientInput.sendKeys(client);
+
+    expect(clientInput.getAttribute('value')).toBe(client);
+  });
+
+  describe('On the calendar', () => {
+    let selectedNumber: ElementFinder,
+        aDayCell: ElementFinder;
+
+    beforeEach(() => {
+      aDayCell = editCraPage.getElementsByText('4').first();
+      aDayCell.click();
+      selectedNumber = editCraPage.getElementsByText('Nombre total de journées :')
+              .first()
+              .element(by.xpath('..//span[@class="highlight"]'));
+    });
+
+    it('should be possible to select days', () => {
+      expect(editCraPage.getADayBadge(aDayCell).getText()).toBe('1\njour');
+    });
+
+    it('It should be possible to select every working day', () => {
+      editCraPage.getElementsByText('Sélectionner les jours ouvrés').first().click();
+
+      expect(selectedNumber.getText()).toBe('21');
```

Comment:
Is the test still going to pass if you run them on a month that does not have 21 working days?

Reply:
no

Reply:
It's a detail, but no need to write twice "it": 
`it('should be possible to...` seems more correct to me than `it('It should be possible to...`